src/handler/auth.ts

```diff
@@ -0,0 +1,36 @@
+import 'dotenv/config';
+import type { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { StatusCodes, getReasonPhrase } from 'http-status-codes';
+import HttpResponse from '../util/httpResponse';
+import {
+  ErrorMissingGrantType, AuthenticatedMessage, ErrorMissingClientID, ErrorMissingClientSecret, ErrorMissingScope,
+} from '../util/authorisationResponses';
+import {
+  containsClientId, containsClientSecret, containsGrantType, containsScope,
+} from '../util/validatorsAuthentication';
+
+// eslint-disable-next-line @typescript-eslint/require-await
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
```

Comment:
is `async` required if you are disabling require-await?

Reply:
I need the async as part of the SAM local chain, this was part of the starter.